src/drivers/uavcan/uavcan_params.c

```diff
@@ -97,6 +97,22 @@ PARAM_DEFINE_FLOAT(UAVCAN_RNG_MIN, 0.3f);
  */
 PARAM_DEFINE_FLOAT(UAVCAN_RNG_MAX, 200.0f);
 
+/**
+ * UAVCAN fuel tank maximum capacity
+ *
+ * This parameter defines the maximum fuel capacity of the vehicle's fuel tank.
+ * Note: Should a dedicated PX4 ECU driver be added in the future, consider transitioning
```

Comment:
I wouldn't put this in a parameter comment, can you either move to a comment in code or drop it?

Reply:
Yep I removed it.